admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -0,0 +1,282 @@
+<!--**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *-->
+<template>
+  <div id="combination-edit-modal">
+    <modal
+      v-if="selectedCombinationId !== null"
+      @close="closeModal"
+    >
+      <template #body>
+        <div
+          class="combination-loading"
+          v-if="loadingCombinationForm"
+        >
+          <div class="spinner" />
+        </div>
+        <iframe
+          ref="iframe"
+          class="combination-iframe"
+          :src="editCombinationUrl"
+          @loadstart="frameLoading"
+          @load="onFrameLoaded"
+          vspace="0"
+          hspace="0"
+          scrolling="auto"
+        />
+      </template>
+
+      <template #footer>
+        <button
+          type="button"
+          class="btn btn-secondary btn-close"
+          @click.prevent.stop="closeModal"
+          aria-label="Close modal"
+          :disabled="submittingCombinationForm"
+        >
+          {{ $t('modal.close') }}
+        </button>
+
+        <button
+          type="button"
+          class="btn btn-outline-secondary"
+          @click.prevent.stop="showPrevious"
+          aria-label="Close modal"
+          :disabled="previousCombinationId === null || submittingCombinationForm"
+        >
+          <i class="material-icons">keyboard_arrow_left</i>
+          {{ $t('modal.previous') }}
+        </button>
+        <button
+          type="button"
+          class="btn btn-outline-secondary"
+          @click.prevent.stop="showNext"
+          aria-label="Close modal"
+          :disabled="nextCombinationId === null || submittingCombinationForm"
+        >
+          {{ $t('modal.next') }}
+          <i class="material-icons">keyboard_arrow_right</i>
+        </button>
+        <button
+          type="button"
+          class="btn btn-primary"
+          @click.prevent.stop="submitForm"
+          aria-label="Close modal"
```

Comment:
```suggestion
          aria-label="Close modal"
```

Use i18n + the label is wrong